src/analysis/retail/monk/windwalker/modules/talents/XuensBattlegear.tsx

```diff
@@ -31,7 +31,7 @@ class XuensBattlegear extends Analyzer {
   constructor(options: Options) {
     super(options);
 
-    this.active = false;
+    this.active = true;
```

Comment:
Instead of setting this to always active, set it conditionally based on whether or not the talent is selected. 

`this.active = this.selectedCombatant.hasTalent(TALENTS_MONK.XUENS_BATTLEGEAR_TALENT);`